src/plugins/intel_gpu/src/graph/impls/ocl_v2/sdpa_micro.cl

```diff
@@ -482,6 +490,25 @@ KERNEL(micro_sdpa)(OPTIONAL_SHAPE_INFO_ARG
         tile_atomic_max_full(
                 S_max_tile, S_max_slm, ugemm_kq_wg_tile_n, sg_j0_kq, 0);
         intel_work_group_barrier_arrive(CLK_LOCAL_MEM_FENCE);
+        #ifdef HAS_SINK_INPUT
+        const int head_idx = get_group_id(1);
+        const float sink_val = convert_float(sink_ptr[head_idx]);
+        float2 sink_val_vec = {sink_val, sink_val};
```

Comment:
@yeonbok This part seems fragile -- if I understand correctly, it only works if `ugemm_kq_sg_tile_n` is either one or two subgroups. Is there a reason the "scalar" path (lines 505-507) doesn't work in all cases?

Reply:
oh I should've generalized this too. Thank you for the comment!